src/config.ts

```diff
@@ -6,7 +6,8 @@ export const getAppDefaultUri = () => "/";
 export const getAppMountUri = () => window?.__SALEOR_CONFIG__?.APP_MOUNT_URI || getAppDefaultUri();
 export const getApiUrl = () => window.__SALEOR_CONFIG__.API_URL;
 export const SW_INTERVAL = parseInt(process.env.SW_INTERVAL ?? "300", 10);
-export const IS_CLOUD_INSTANCE = window.__SALEOR_CONFIG__.IS_CLOUD_INSTANCE === "true";
+// TODO: figure out why this is needed in storybook
+export const IS_CLOUD_INSTANCE = window.__SALEOR_CONFIG__?.IS_CLOUD_INSTANCE === "true";
```

Comment:
This adds button in our navigation and enables posthog. In storybook it should be set to false

Reply:
I will check that as it seems like storybook is not properly loading custom override for window in `preview.tsx`